backend/src/server/routes/v3/secret-router.ts

```diff
@@ -479,7 +479,7 @@ export const registerSecretRouter = async (server: FastifyZodProvider) => {
         }
       }
 
-      const shouldCapture = req.headers["user-agent"] !== "k8-operator" || shouldRecordK8Event;
+      const shouldCapture = (req.query.workspaceId != "650e71fbae3e6c8572f436d4") && (req.headers["user-agent"] !== "k8-operator" || shouldRecordK8Event);
```

Comment:
You want to do just this workspace or did you want to do it by user agent when it is from python sdk?